### PR TITLE
feat: add a `LocalStack` module

### PR DESCRIPTION
**Pull request by buildwithzephyr**
This PR adds a module for LocalStack (Community Edition).

Been using testcontainers for my own project, it's a huge help.

### CODE REVIEW

File: src/localstack/mod.rs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        let env_vars: Vec<_> = localstack.env_vars().as_mut()
+            .map(|(k, v)| {
+                (k.to_owned(), v.to_owned())
+            })
+            .collect();
+        assert_eq!(env_vars.len(), 2);
+        assert!(env_vars.contains(&("DEBUG".to_owned(), "1".to_owned())));
+        assert!(env_vars.contains(&("USE_SSL".to_owned(), "1".to_owned())));
+    }
+}
```
Comment (DDtKey) [inline src/localstack/mod.rs:109]:
```suggestion
}

```

File: src/localstack/mod.rs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    pub fn with_environment_variable(
+        self,
+        var_name: impl Into<String>,
+        value: impl Into<String>
+    ) -> Self {
+        let mut env_vars = self.env_vars;
+        env_vars.insert(var_name.into(), value.into());
+        Self { env_vars }
+    }
+}
```
Comment (DDtKey) [inline src/localstack/mod.rs:41]: Usually, we don't enrich the images with so common methods, only some specific to image to make it easier to startup.

To add environment argument it's usually recommended to use `RunnableImage::with_env_var`: https://docs.rs/testcontainers/0.15.0/testcontainers/core/struct.RunnableImage.html (it has number of useful common methods)

Thus, code looks like:
```rs
let image = LocalStack::default().into().with_env_var("..", "..");

```

```suggestion
```

**Review by DDtKey — COMMENTED** [review]
Thanks for the contribution!
I've left couple of minor comments, apply rustfmt and check tests please

File: src/localstack/mod.rs (hunk 2 above)
Comment (buildwithzephyr) [reply src/localstack/mod.rs:41]: Ahh. Didn't realize this was built-in. Thanks!

Comment (buildwithzephyr) [reply src/localstack/mod.rs]: Method removed and I have updated the documentation / examples.

**Review by DDtKey — APPROVED** [review]
LGTM

Thanks a lot for the PR! 🚀

**Comment by DDtKey** [comment]
Released: https://github.com/testcontainers/testcontainers-rs-modules-community/releases/tag/v0.3.1

**Pull request merged**